Rocket.Chat.SDKTests/Managers/LivechatManagerTests.swift

```diff
@@ -12,6 +12,7 @@ import SwiftyJSON
 
 class LivechatManagerTests: XCTestCase {
 
+    // swiftlint:disable:next force_unwrapping
     let socket = WebSocketMock(url: URL(string: "http://doesnt.matter")!)
```

Comment:
Can't you do it in a `guard` or `if let`? Let's avoid disabling swxftlint features.